### PR TITLE
add extra configuration for Console JWT auth

### PR DESCRIPTION
**Pull request by ericsyh**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml [hunk 1]
```diff
@@ -278,7 +278,7 @@ spec:
           {{- end }}
           {{- if eq .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_TOKEN_MODE "SECRET" }}
           - key: SECRETKEY
-            path: token/secret.key
+            path: {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH }}
```
Comment (tuteng) [inline charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml:281]:
```
{{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH | default "token/secret.key" }}
```

File: charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml [hunk 2]
```diff
@@ -205,7 +205,7 @@ spec:
           - name: JWT_BROKER_TOKEN_MODE
             value: SECRET
           - name: JWT_BROKER_SECRET_KEY_PATH
-            value: /pulsar-manager/keys/token/secret.key
+            value: {{ "/pulsar-manager/keys/" }} + {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH }}
```
Comment (tuteng) [inline charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml:208]: {{ "/pulsar-manager/keys/" }} + {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH | default "token/secret.key" }}

File: charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml [hunk 3]
```diff
@@ -211,7 +211,7 @@ spec:
           - name: JWT_BROKER_TOKEN_MODE
             value: SECRET
           - name: JWT_BROKER_SECRET_KEY_PATH
-            value: /pulsar-manager/keys/token/secret.key
+            value: {{ "/pulsar-manager/keys/" }} + {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH }}
```
Comment (tuteng) [inline charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml:214]: {{ "/pulsar-manager/keys/" }} + {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH | default "token/secret.key" }}

File: charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml [hunk 4]
```diff
@@ -288,7 +288,7 @@ spec:
           {{- end }}
           {{- if eq .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_TOKEN_MODE "SECRET" }}
           - key: SECRETKEY
-            path: token/secret.key
+            path: {{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH }}
```
Comment (tuteng) [inline charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml:291]:
```
{{ .Values.streamnative_console.login.sso.pulsarJwt.config.JWT_BROKER_SECRET_KEY_PATH | default "token/secret.key" }}
```

Comment (ericsyh) [reply charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml]: fix by https://github.com/streamnative/charts/pull/1191/commits/5a99886a990da4a1c18979723af826964745ea39

File: charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml (hunk 3 above)
Comment (ericsyh) [reply charts/sn-platform/templates/streamnative-console/streamnative-console-statefulset.yaml]: fix by https://github.com/streamnative/charts/pull/1191/commits/5a99886a990da4a1c18979723af826964745ea39

File: charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml (hunk 2 above)
Comment (ericsyh) [reply charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml]: fix by https://github.com/streamnative/charts/pull/1191/commits/5a99886a990da4a1c18979723af826964745ea39

File: charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml (hunk 1 above)
Comment (ericsyh) [reply charts/sn-platform-slim/templates/streamnative-console/streamnative-console-statefulset.yaml]: fix by https://github.com/streamnative/charts/pull/1191/commits/5a99886a990da4a1c18979723af826964745ea39

**Review by tuteng — APPROVED** [review]

**Pull request merged**